docs/hybrid_applications.md

```diff
@@ -0,0 +1,71 @@
+# Hybrid applications
+
+## Overview
+
+You can use Datadog on the same application both from the React Native and the native sides.
+
+RUM Events will be reported as coming from the same application and the same source in Datadog RUM.
+
+## Constraints
+
+A few constraints to keep in mind:
+
+1. You cannot initialize more than 1 instance of the native `DatadogSDK` as it uses a singleton pattern on the native side. That means that once you initialize the native SDK on the native side or on the RN side (by calling `DdSdkReactNative.initialize`) it is configured for both.
```

Comment:
> That means that once you initialize the native SDK on the native side (....) it is configured for both.
If my current understanding of the RN SDK is correct, this seems misleading as if someone initializes the SDK on the native side only, then RN functionality will remain deactivated.
Please correct me if I'm wrong.

Reply:
Yes, I'm going to go more into details here